external-import/mandiant/src/connector/__main__.py

```diff
@@ -7,6 +7,7 @@
     mandiantConnector = Mandiant()
     while True:
         mandiantConnector.run()
-except Exception:
+except Exception as err:
+    print(err)
     time.sleep(10)
```

Comment:
Not a big deal and mostly out-of-scope, but that sounds like this shouldn't be handled inside the connector, but on what is managing its process (systemd, supervisord etc.).

A lot of connectors have this.

Reply:
This is a little fix for this connector. 
I go along with you that we may have another process to catch the exception.
Let's talk about it :)

Reply:
@gileri Following our discussion, we will add a track trace to better log exception :)